### PR TITLE
Commonsdivision atom - fix app style and make MP lists optional

### PR DESCRIPTION
**Pull request by tomrf1**
- Overrides some border styling in the app
- Adds a flag to disable the rendering of MP lists. This doesn't work properly on the app, and needs a better solution. For now we'll just hide it.

### CODE REVIEW

File: core/src/main/scala/Configuration.scala [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 
 sealed trait Configuration
 
 sealed trait NilConfiguration extends Configuration
 case object NilConfiguration extends NilConfiguration
 
 final case class ArticleConfiguration(
-  ajaxUrl: String
+  ajaxUrl: String,
+  commonsDivisionConfiguration: CommonsDivisionConfiguration = CommonsDivisionConfiguration()
```
Comment (regiskuckaertz) [inline core/src/main/scala/Configuration.scala]: I'd prefer it for now if the config was completely transparent and there was not default value.

Comment (regiskuckaertz) [reply core/src/main/scala/Configuration.scala]: Can we use the same spelling as the atom name itself? So just `commonsdivision`, all lowercase?

Comment (regiskuckaertz) [reply core/src/main/scala/Configuration.scala]: ... perhaps in the future properties like `ajaxUrl` should be stuck in a `global` subconfig object.

Comment (tomrf1) [reply core/src/main/scala/Configuration.scala]: I can't believe I didn't make it all lowercase after all the trouble that has caused previously!

**Pull request merged**